libraries/botbuilder-dialogs/src/dialogManager.ts

```diff
@@ -334,6 +336,24 @@ export class DialogManager extends Configurable {
         return turnResult;
     }
 
+    /**
+     * @private
```

Comment:
JSDoc comments aren't required for private methods, though feel free. I've just used the `//` comment syntax on private methods for internal-only documentation.